### PR TITLE
[FE-229] improve shard rebalancing UI

### PR DESCRIPTION
**Pull request by pluma4345**
### Scope & Purpose

- [ ] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [x] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.11: *(Please link PR)*
  - [ ] Backport for 3.10: *(Please link PR)*
  - [ ] Backport for 3.9: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [x] GitHub issue / Jira ticket: FE-229
- [ ] Design document: 

image

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  try {
+    const res = await route.put("cluster/rebalance", {
+      version: 1,
+      ...opts
+    });
+    const result = res.body.result;
+    if (result.moves.length === 0) {
+      window.arangoHelper.arangoNotification('No move shard operations were performed.');
+    } else {
+      window.arangoHelper.arangoNotification('Rebalanced shards with ' + result.moves.length + ' move operation(s).');
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx:28]: maybe we can prefer template strings here?

```suggestion
      window.arangoHelper.arangoNotification(`Rebalanced shards with ${result.moves.length} move operation(s).`);
```

File: js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            marginBottom: "unset"
+          }
+        }}
+      />
+        <Box width="100%" padding="5" background="white">
+          <Box fontSize={"lg"}>
+            Shard rebalancing
+          </Box>
+          <Box
+            display={"grid"}
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx:69]: Can use Grid from chakra
```suggestion
          <Grid
```

File: js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      <Form>
+      <Global
+        styles={{
+          "label": {
+            marginBottom: "unset"
+          }
+        }}
+      />
+        <Box width="100%" padding="5" background="white">
+          <Box fontSize={"lg"}>
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx:65]: Can use Text from Chakra
```suggestion
          <Text fontSize={"lg"}>
```

File: js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js [hunk 4]
```diff
@@ -517,11 +483,14 @@
           return;
         }
 
-        if (this.shardDistributionView) {
-          this.shardDistributionView.remove();
-        }
-        this.shardDistributionView = new window.ShardDistributionView({});
-        this.shardDistributionView.render();
+        ReactDOM.render(
+          React.createElement(ShardDistributionReactView, {
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js:487]:
```suggestion
          React.createElement(window.ShardDistributionReactView, {
```

File: js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/shards/distribution/RebalanceShards.tsx:1]: as discussed, let's ensure we do prettier formatting for all new files

**Review by palashkaria — CHANGES_REQUESTED** [review]

File: js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/AddIndexForm.tsx [hunk 5]
```diff
@@ -6,10 +6,10 @@ import { IndexType } from "../useFetchIndices";
 import { FulltextIndexForm } from "./FulltextIndexForm";
 import { GeoIndexForm } from "./GeoIndexForm";
 import { IndexInfoTooltip } from "./IndexInfoTooltip";
-import { InvertedIndexFormWrap } from "./invertedIndex/InvertedIndexFormWrap";
 import { PersistentIndexForm } from "./PersistentIndexForm";
 import { TTLIndexForm } from "./TTLIndexForm";
 import { ZKDIndexForm } from "./ZKDIndexForm";
+import { InvertedIndexFormWrap } from "./invertedIndex/InvertedIndexFormWrap";
```
[the remark this answers is not in the document]
Comment (palashkaria) [reply js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/AddIndexForm.tsx:12]: I mostly run organize imports, possible that I could've missed it here. But even then - "Z" should be last, and "I" should be above "P"? (as it already was?)

**Review by ajurdak — APPROVED** [review]
I reviewed the code as well as the UI itself. Great work.
LGTM.

**Review by palashkaria — COMMENTED** [review]
LGTM!

**Review by palashkaria — APPROVED** [review]

**Pull request merged**